sauceplugin-agent/src/main/java/com/saucelabs/teamcity/NewSauceConnectManager.java

```diff
@@ -0,0 +1,48 @@
+package com.saucelabs.teamcity;
+
+import com.saucelabs.ci.sauceconnect.SauceConnectManager;
+import com.saucelabs.ci.sauceconnect.SauceTunnelManager;
+import jetbrains.buildServer.log.Loggers;
+
+import java.io.*;
+import java.util.function.Consumer;
+
+/**
+ * Handles launching Sauce Connect (binary executable).
+ *
+ * @author Ross Rowe
+ */
+public class NewSauceConnectManager extends SauceConnectManager
```

Comment:
This is going to be removed once https://github.com/saucelabs/ci-sauce/pull/308 is merged.